### ISSUE TITLE
The type of lineWidth should be number at NavigatorHandlesOptions

### ISSUE THREAD

**Issue by bill-youngdi** [highcharts/highcharts#16457]
#### Expected behaviour
lineWidth's type should be `number`

#### Actual behaviour
lineWidth's type is `object`

I think the issue is that the dist project doesn't update the `highcharts.d.ts` file. Because I check the source code here lineWidth's type is number
https://github.com/highcharts/highcharts/blob/a6e375c7fa92fd8c816e648dc9f3106da1b2f260/ts/Core/Navigator.ts

**Comment by KacperMadej** [comment]
Hi @bill-youngdi ,

Thank you for reporting the problem.


_Internal note:_
tl;dr delete the line https://github.com/highcharts/highcharts/blob/a6e375c7fa92fd8c816e648dc9f3106da1b2f260/ts/Core/Navigator.ts#L442

---

.d.ts on dist has:
```
    /**
     * (Highstock, Gantt) The width for the handle border and the stripes
     * inside.
     */
    lineWidth?: object;
```
and the code defining seems to be different here:
https://github.com/highcharts/highcharts/blob/a6e375c7fa92fd8c816e648dc9f3106da1b2f260/ts/Core/Navigator.ts#L435-L444

API reference is missing the type and default value:
https://api.highcharts.com/highstock/navigator.handles.lineWidth

This is because `apioption` was used and later type is or default value is neither guessed nor used from the code below the comment in the Navigator.ts file.
